src/views/Player.vue

```diff
@@ -396,10 +420,9 @@ export default {
           progressBar.firstChild.disabled = true;
 
         // initializing the retention array with zeros
-        this.retention = Array(this.player.duration).fill(0);
-        this.submitted = Array(this.plioQuestions.length).fill(0);
-        this.skipped = Array(this.plioQuestions.length).fill(0);
-        this.revised = Array(this.plioQuestions.length).fill(0);
```

Comment:
Removed these three lines as this was related to an older implementation and were redundant now.